File: 0657-robot-return-to-origin/0657-robot-return-to-origin.java
class Solution {
    public boolean judgeCircle(String moves) {
        int count_v = 0, count_h = 0;
        for (int i = 0; i < moves.length(); i++) {
            char ch = moves.charAt(i);
            if (ch == 'U') count_v++;
            else if (ch == 'D') count_v--;
            else if (ch == 'R') count_h++;
            else count_h--;
        }
        return count_v == 0 && count_h == 0;
    }
}

File: 0657-robot-return-to-origin/0657-robot-return-to-origin.cpp
class Solution {
public:
    bool judgeCircle(string moves) {
        int count_v=0,count_h=0;
        for(int i=0;i<moves.length();i++){
            if(moves[i]=='U') count_v++;
            else if(moves[i]=='D') count_v--;
            else if(moves[i]=='R') count_h++;
            else count_h--;
        }
        if(count_v==0 && count_h==0) return true;
        return false;
    }
};